epochtalk/epochtalk
# Banning

Test Procedure:

1) Banning a user should now add the banned role

2) Removing banned role should remove ban

3) Blacklist an ip and spoof ip in the blacklist plugin to test
